src/Core/Framework/Test/DataAbstractionLayer/Field/FloatFieldTest.php

```diff
@@ -0,0 +1,193 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Test\DataAbstractionLayer\Field;
+
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\Flag\ApiAware;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\Flag\Required;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\FloatField;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\IntField;
+use Shopware\Core\Framework\DataAbstractionLayer\FieldSerializer\FloatFieldSerializer;
+use Shopware\Core\Framework\DataAbstractionLayer\FieldSerializer\IntFieldSerializer;
+use Shopware\Core\Framework\DataAbstractionLayer\Write\DataStack\KeyValuePair;
+use Shopware\Core\Framework\DataAbstractionLayer\Write\EntityExistence;
+use Shopware\Core\Framework\DataAbstractionLayer\Write\WriteParameterBag;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelTestBehaviour;
+use Shopware\Core\Framework\Validation\WriteConstraintViolationException;
+
+/**
+ * @group FloatFieldTest
+ * @internal
+ */
+class FloatFieldTest extends TestCase
+{
+    use KernelTestBehaviour;
+
+    public function testFloatFieldSerializerNullValue(): void
+    {
+        $serializer = $this->getContainer()->get(FloatFieldSerializer::class);
```

Comment:
Can you make a unit test? Without container and move to tests/unit/

Reply:
@shyim but the IntFieldSerializer is test that way, i just reused the IntFieldTest